seirmo/apps/simulation_app.py

```diff
@@ -0,0 +1,76 @@
+#
+# This file is part of seirmo (https://github.com/SABS-R3-Epidemiology/seirmo/)
+# which is released under the BSD 3-clause license. See accompanying LICENSE.md
+# for copyright notice and full license details.
+#
+
+import pandas as pd
+import seirmo as se
+
+import dash
+import dash_bootstrap_components as dbc
+import dash_core_components as dcc
+
+
+class SimulationApp(object):
+    """SimulationApp Class:
+
+    Creates the SEIR model simulation app.
+
+    """
+
+    def __init__(self):
+        super(SimulationApp, self).__init__()
+
+        self.IncidenceNumberPlot = se.IncidenceNumberPlot()
```

Comment:
I would call this self._fig (variables always small case and as concise as possible.